src/components/PostDetails.tsx

```diff
@@ -1,117 +1,101 @@
 import React from 'react';
 import { Loader } from './Loader';
+import { CommentsList } from './CommentsList';
 import { NewCommentForm } from './NewCommentForm';
+import { Comment } from '../types/Comment';
+import { Reset } from '../types/Reset';
 
-export const PostDetails: React.FC = () => {
+type Props = {
+  comments: Comment[] | null,
+  postTitle?: string,
+  postBody?: string,
+  postId?: number,
+  isWrongPostResponse: boolean,
+  isPostLoading: boolean,
+  isCommentForm: boolean,
+  isLoadingNewComment: boolean,
+  isDangerSubmit: boolean,
+  isWarningUpdate: boolean,
+  setIsCommentForm: (value: boolean) => void,
+  onHandleFormSubmit: (
+    event: React.FormEvent<HTMLFormElement>,
+    reset: (inputType: Reset) => void,
+    name: string,
+    email: string,
+    body: string,
+    postId?: number,
+  ) => void,
+  canselDangerInput: () => void,
+  onHandleDeleteComment: (id: number) => void,
+};
+
+export const PostDetails: React.FC<Props> = ({
+  comments,
+  postTitle,
+  postBody,
+  isWrongPostResponse,
+  isPostLoading,
+  postId,
+  isCommentForm,
+  isLoadingNewComment,
+  isDangerSubmit,
+  isWarningUpdate,
+  onHandleFormSubmit,
+  setIsCommentForm,
+  canselDangerInput,
+  onHandleDeleteComment,
+}) => {
   return (
     <div className="content" data-cy="PostDetails">
       <div className="content" data-cy="PostDetails">
         <div className="block">
           <h2 data-cy="PostTitle">
-            #18: voluptate et itaque vero tempora molestiae
+            {`${postId}: ${postTitle}`}
```

Comment:
```suggestion
            {`#${postId}: ${postTitle}`}
```
![image](https://github.com/mate-academy/react_dynamic-list-of-posts/assets/96409563/3d694c86-d4d4-4e7a-8a9d-1597e95b7947)